packages/framework/presence/package.json

```diff
@@ -134,6 +129,7 @@
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
 		"@biomejs/biome": "~1.9.3",
+		"@fluid-experimental/presence": "^2.5",
```

Comment:
I think we want to rename like other priors for typetest, no? Otherwise, unused.
I think we want `"@fluidframework/presence-previous": "npm:@fluid-experimental/presence@2.5.0"`